src/join/gql/addUserMutation.ts

```diff
@@ -0,0 +1,12 @@
+import gql from "graphql-tag";
+
+export const AddUserGQL = gql`
+  mutation AddUser($campaignId: String!) {
```

Comment:
I think this might be better named 'joinCampaign' as we don't actually have the ability to specify the user we're adding. It's an action taken by an authenticated user enabled by the headers we send. There might be scope for an 'addUser' mutation but I'd expect that to take a userId as an input as well

Reply:
Agreed! I’ll rename the mutation